### DISCUSSION TITLE
Enregistrer des ressources sur Mes Cantines

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1467] [Idées]
**Besoin :**
Possibilité d'enregistrer des ressources – par exemple, une info-lettre ou les objectifs alimentaires de l'année – sur l'espace Mes Cantine. Besoin exprimé par une utilisatrice qui travaille pour un Département.

**Intérêt :**
Faire de Ma Cantine LA plateforme qui rassemble toutes les ressources et tous les échanges entre une collectivité territoriale et les chefs de cuisine ou gestionnaires sur le terrain. Concentration des données à un seul endroit.

**Remarque perso :**
Cette demande est hyper encourageante! Cela signifie que certains de nos utilisateurs voient déjà Ma Cantine comme le seul outil qu'ils utiliseront à terme pour le sujet de la restauration collective. 🙂

**Comment by alemangui** [comment]
Intéressant et encourageant en effet ! Je me pose de questions sur la forme que cela aurait - on parle de pouvoir télécharger dans _ma cantine_ des documents/fichiers qui potentiellement sont à l'extérieur ? Ou bien d'accéder plus facilement aux ressources déjà dans _ma cantine_ (par ex la doc, les pages de description des mesures, etc) ?

**Reply by ghost** [reply]
L'utilisatrice envisageait de pouvoir téléverser dans Ma Cantine des documents qui viennent de l'extérieur dans un espace personnel qu'elle partagerait avec les autres gestionnaires de la cantine.
